.drone.yml

```diff
@@ -1,3 +1,112 @@
+---
```

Comment:
Please drop the related commit, see below how to rebase against the master branch.